src/main/java/codeu/bot/ActionMatcher.java

```diff
@@ -380,13 +526,33 @@ public void matchAction(String input, String username) throws IOException {
         System.out.println(commandPOS);
         System.out.println(nerTags);
         System.out.println(timexAnnotationsAll);
+        System.out.println(commandTokensLemmas);
 
         matchSet();
-        matchSendMessage();
-        matchCreateConversation();
-        matchGet();
-        matchSummarize();
-        return;
+        if (!actionMatched) {
```

Comment:
can we update this to be less nested?

Reply:
An else if should be able to do this job. I did something similar in my nlp pull request